charts/gsp-cluster/templates/02-gsp-system/egress-safelist.yaml

```diff
@@ -0,0 +1,15 @@
+{{- range $i, $egress := .Values.egressSafelist }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: {{ $.Release.Name }}-egress-safelist-{{ $i }}
```

Comment:
Naming based on the index feels weird and potentially dangerous if someone adds something mid way through the list, or removes anything at all

Reply:
Yeah, guess our options are either using a map:

```
egressSafelist:
  integrationHub:
    hosts: ["www.integration.signin.service.gov.uk"]
    ports:
    - name: verify-integration-https
      number: 443
      protocol: TCP
    location: MESH_EXTERNAL
    resolution: DNS
  productionHub:
    hosts: ["www.signin.service.gov.uk"]
    ports:
    - name: verify-https
      number: 443
      protocol: TCP
    location: MESH_EXTERNAL
    resolution: DNS
```

or adding the `name` back

```
egressSafelist:
- name: integration-hub
  service:
    hosts: ["www.integration.signin.service.gov.uk"]
    ports:
    - name: verify-integration-https
      number: 443
      protocol: TCP
    location: MESH_EXTERNAL
    resolution: DNS
- name: production-hub
  service:
    hosts: ["www.signin.service.gov.uk"]
    ports:
    - name: verify-https
      number: 443
      protocol: TCP
    location: MESH_EXTERNAL
    resolution: DNS
```

Reply:
Resolved in 1f930af.